src/main/java/de/chojo/repbot/commands/bot/BotAdmin.java

```diff
@@ -53,6 +54,9 @@ public BotAdmin(Guilds guilds, Configuration configuration, Statistic statistics
                 .subCommand(SubCommand.of("search", "Search for guilds")
                         .handler(new Search())
                         .argument(Argument.text("term", "Search term").asRequired()))
+                .subCommand(SubCommand.of("invalidate_cache", "Invalidates cached data for the guild")
+                        .handler(new InvalidateCache(guilds))
+                        .argument(Argument.text("guild", "guild id").asRequired()))
```

Comment:
Should there be a way to invalidate the cache for all guilds?

Reply:
That mainly for debug purposes when I change a value in the database and wont those to be reflected on the guild.  If I want to invalidate all guilds a restart would be the better option anyway I think.